include/http/libcurl_http_client.h

```diff
@@ -127,6 +127,26 @@ namespace azure {  namespace storage_lite {
             check_code(curl_easy_setopt(m_curl, CURLOPT_READDATA, this));
         }
 
+        void set_input_stream_length(size_t streamlen)
+        {
+            m_input_stream_len=streamlen;
+        }
+
+        void set_is_stream_length(void)
+        {
+            m_input_stream_len_known=true;
+        }
+
+        bool is_stream_length(void)
+        {
+            return m_input_stream_len_known;
+        }
```

Comment:
Minor: the mismatch name in getter/setter and class member isn't a good practice. Please align them.